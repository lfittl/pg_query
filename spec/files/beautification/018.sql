with x as (Select relkind, count(*) FROM pg_class group BY 1) SELECT count(*) FROM x;
