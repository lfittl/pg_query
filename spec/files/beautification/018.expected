WITH x AS (
  SELECT
    "relkind",
    count(*)
  FROM
    "pg_class"
  GROUP BY
    1
)
SELECT
  count(*)
FROM
  "x"
